mods/pageviews.js

```diff
@@ -320,9 +305,25 @@ PJVS.prototype.pageviewsForProjects = function(restbase, req) {
             }
         }
 
-    });
+    }).catch(notFoundCatcher);
 
-    return dataRequest.then(normalizeResponse);
+    return dataRequest.then(normalizeResponse).then(function(res) {
+        if (res.body.items) {
+            res.body.items.forEach(function(item) {
+                // prefer the v column if it's loaded
+                if (item.hasOwnProperty('v') && item.v !== null) {
```

Comment:
Heh, no, my point was exactly that you don't even need to test for `v`, as it will be there for sure (even though as you are writing the code it doesn't). So `if (item.v !== null)` will be enough.